### DISCUSSION TITLE
Simple Summarize function for selection/search

### DISCUSSION THREAD

**Discussion by MartinThorn** [hajkmap/Hajk#1476] [Ideas]
Maybe its possible somehow only I did not find it yet.? Did a search on summarize function and did not find here either... Anyhow I miss a quite standard "Sumarize function tool" for fields that are in numeric form format. In that way you could have some interesting fields in the database and the usera can do some analyzing them selves. I know you can export a seach result to Excel, but it a bit tricky aswell as its the wrong format in excel so there you first have to transform the column values to numbers to be able to summarize a field.   
![image](https://github.com/hajkmap/Hajk/assets/150000798/38fd6f8f-7f10-4ff7-946c-dd9abfa759ad)

**Comment by jacobwod** [comment]
Are you referring to the ability to sum up something specific, e.g. features drawn using the Sketch tool? Or Measurer perhaps?
![Skärmavbild 2024-02-14 kl  10 43 07](https://github.com/hajkmap/Hajk/assets/110222/e02d3206-7845-4f5c-a4b5-73a70780667d)

Or is it the infoclick functionality and the results shown in the infoclick window that you'd like to display as a table and summarise?

**Comment by MartinThorn** [comment]
I am thinking more of the infocklick (or search function) so you could for example "search with radius" or rectangle etc and it would select features and also shown in a table form in a infowindow, if possible a simplified Summarized window like "Σ" do.  Usally total hits, max/min, mean and in my case "sum" of the numeric fields hits.

**Comment by jacobwod** [comment]
OK, I see. A kind of infoclick that does not trigger on click (like it does currently) but rather allows the user to select multiple features by some means (e.g. shift+click or draw a polygon/circle). Next, it would collect grab the results and show features in map and the infoclick window. In addition, some kind of aggregate function would be run to sum up _some_ attributes, perhaps from the geometries (or some other attribute, that would have to be specified in Admin). Next, the results of the aggregate function would be presented in a table. I hope I got it right this time.

Anyway: no, there's no such functionality implemented currently and I am not aware of anyone working on it either. Contributions are welcome!

**Reply by MartinThorn** [reply]
Yes exactly so, we are moving from historical desktop solutions as "GIS viewers" and usally its possible to do select/aggregate in the desktop viewers, but I guess one could redirect those users to use QGIS for that instead. Thanks/M
